pyfar/dsp/dsp.py

```diff
@@ -1487,4 +1494,354 @@ def decibel(signal, domain='freq', log_prefix=None, log_reference=1):
+    signal: Signal, TimeData, FrequencyData
+        Input signal.
+    domain: string
+        Determines whih data is used to compute the `reference` value.
+
+        ``'time'``
+           Use the time data ``signal.time``.
+        ``'freq'``
+          Use the frequency data ``signal.freq``. Note that the data with
+          FFT normalization is used (cf.
+          :py:mod:`FFT concepts <pyfar._concepts.fft>`).
+
+        The default is ``'time'``.
+    operation: string, optional
+        Operation to compute the channel-wise `reference` value using the data
+        according to `domain`.
+
+        ``'max'``
+            Compute the maximum absolute value per channel.
+        ``'mean'``
+            Compute the mean absolute values per channel.
+        ``'energy'``
+            Compute the energy per channel using :py:func:`~pyfar.dsp.energy`.
+        ``'power'``
+            Compute the power per channel using :py:func:`~pyfar.dsp.power`.
+        ``'rms'``
+            Compute the RMS per channel using :py:func:`~pyfar.dsp.rms`.
+
+        The default is ``'max'``.
+    channel_handling: string, optional
+        Define how channel-wise `reference` values are handeled for multi-
+        channel signals. This parameter does not affect single-channel signals.
+
+        ``'each'``
+            Separate normalization of each channel.
+        ``'max'``
+            Normalize to the maximum `reference` value across channels.
+        ``'min'``
+            Normalize to the minimum `reference` value across channels.
+        ``'mean'``
+            Normalize to the mean `reference` value across the channels.
+
+       The default is ``'each'``.
+    target: scalar, array
+        The target to which the signal is normalized. Can be a scalar or an
+        array. In the latter case the shape of `target` must be broadcastable
+        to ``signal.cshape``. The default is ``1``.
+    limits: tuple, array_like
+        Restrict the time or frequency range that is used to compute the
+        `reference`value. Two element tuple specifying upper and lower limit
+        according to `domain` and `unit`. A `None` element means no upper or
+        lower limitation. The default ``(None, None)`` uses the entire signal.
+    unit: string, optional
+        Unit of `limits`.
+
+        ``'s'``
+            Set limits in seconds in case of time domain normalization. Uses
+            :py:class:`~signal.find_nearest_time` to find the limits.
+        ``'Hz'``
+            Set limits in hertz in case of frequency domain normalization. Uses
+            :py:class:`~signal.find_nearest_frequency`
+
+        The default ``None`` assumes that `limits` is given in samples in case
+        of time domain normalization and in bins in case of frequency domain
+        normalization.
+    return_reference: bool
+        If ``return_reference=True``, the function also returns the `reference`
+        values described above. The default is ``False``.
```

Comment:
I'd suggest to the shape clear, e.g.,
```suggestion
        values for the channels. The default is ``False``.
```